### PR TITLE
New: Add Windows testing support

### PR DESCRIPTION
**Pull request by stramel**
Start of #236 Possibly fixes? Tests seem to pass?

Link to passing tests: https://ci.appveyor.com/project/stramel/polymer-modulizer/build/26

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,8 +14,8 @@
   "scripts": {
     "build": "tsc",
     "build:fixtures": "npm run build && node lib/tools/update-fixtures",
-    "test:full": "npm run build && mocha $(find lib/test -name *_test.js) && npm run lint",
-    "test:fast": "npm run build && mocha $(find lib/test/unit -name *_test.js)",
+    "test:full": "npm run build && bash test/run-mocha.sh && npm run lint",
```
Comment (FredKSchott) [inline package.json]: I use bash on my machine so I can't test that this breaks on others, but change to `./test/run-mocha.sh` to support non-bash environments?

File: src/test/fixtures/fixtures_test.ts [hunk 2]
```diff
@@ -142,7 +140,7 @@ suite('Fixtures', () => {
         // 2. Compare the generated output to the expected conversion.
         //    Output the diff & fail if any differences are encountered.
         const diffResult = dircompare.compareSync(
-            fixtureResultDir, fixtureExpectedDir, {compareSize: true});
+            fixtureResultDir, fixtureExpectedDir, {compareSize: false});
```
Comment (FredKSchott) [inline src/test/fixtures/fixtures_test.ts]: why can't we compare sizes?

File: package.json (hunk 1 above)
Comment (stramel) [reply package.json]: I can change it to that. I was basing it off of lighthouse which is what I used as reference to getting mocha to run. https://github.com/GoogleChrome/lighthouse/blob/master/package.json#L32

File: src/test/fixtures/fixtures_test.ts (hunk 2 above)
Comment (stramel) [reply src/test/fixtures/fixtures_test.ts]: I don't remember exactly, it was one of the things i disabled initially. I will double check and try to get you a solid reason.

File: package.json (hunk 1 above)
Comment (stramel) [reply package.json]:
```
stramel@DESKTOP-6G41VHH MINGW64 ~/Projects/GitHub/polymer-modulizer (ms/test-windows)
$ npm test

> polymer-modulizer@0.3.0 test C:\Users\stramel\Projects\GitHub\polymer-modulizer
> npm run test:fast && npm run lint


> polymer-modulizer@0.3.0 test:fast C:\Users\stramel\Projects\GitHub\polymer-modulizer
> npm run build && ./test/run-mocha.sh unit


> polymer-modulizer@0.3.0 build C:\Users\stramel\Projects\GitHub\polymer-modulizer
> tsc

'.' is not recognized as an internal or external command,
```

File: src/test/fixtures/fixtures_test.ts (hunk 2 above)
Comment (stramel) [reply src/test/fixtures/fixtures_test.ts]: image

Looking into this more.

Comment (stramel) [reply src/test/fixtures/fixtures_test.ts]: Looks like generated is generating `\r\n` and expected is `\n`.

**Comment by stramel** [comment]
@FredKSchott So other than line-endings everything else was fine. I updated it to handle line-endings to properly match based on OS and compareSize now works as well.

**Comment by stramel** [comment]
NOTE: template string literal line breaks and stringified json use `\n`.

**Comment by stramel** [comment]
@FredKSchott Can you take another look?

File: package.json (hunk 1 above)
Comment (FredKSchott) [reply package.json]: huh, I thought npm handled that on windows. Okay, bash is fine for now, if anyone is on a shell that can't run that we can cross that bridge when we come to it.

**Comment by FredKSchott** [comment]
Awesome, really excited to have this up and running. I'll get appveyor setup so we can test

**Comment by FredKSchott** [comment]
hmm, looks like its already set up. Maybe just needs to merge

**Pull request merged**